app/models/spree/working_hour.rb

```diff
@@ -0,0 +1,14 @@
+class Spree::WorkingHour < ApplicationRecord
+  belongs_to :working_day
+  validates :start_time, :end_time, presence: true
+  validate :validate_interval
+  default_scope { order(start_time: :asc, end_time: :asc) }
+
+  def validate_interval
```

Comment:
how are you validating overlaps? in the same working day?

Reply:
For now there is no validation for overlaps.